Flow.Launcher/Helper/FileExplorerHelper.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.IO;
+
+namespace Flow.Launcher.Helper
+{
+    public class FileExplorerHelper
```

Comment:
The other function that *might* be interesting would be getting all open file explorer paths (sorted by depth/zindex). (See also https://github.com/jjw24/Wox.Plugin.Runner/pull/27 )
Thoughts?

Reply:
I agree, maybe we should return the paths sorted by depth/zindex (since we will need to get all of them). The overhead can be ignored.